templates/individual_response/interstitial.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'layouts/_base.html' %}
+{% import 'macros/helpers.html' as helpers %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/summary/_macro.njk" import onsSummary %}
+{% from "components/button/_macro.njk" import onsButton %}
+
+{% set save_on_signout = true %}
+
+
+{% block main %}
+    <button type="submit" class="u-d-no" hidden></button>
+    <div class="u-mb-s">
+      <a {{ helpers.track('click', 'Navigation', 'Previous link click', position )}} href="{{ previous_location_url }}" id="top-previous">{{ _("Previous") }}</a>
+    </div>
+    <h1 class="u-mt-m">{{_("If you can't answer questions for others in your household") }}</h1>
+    <p>{{_("You can ask the people you live with to answer their own questions by sharing the household access code with them.") }}</p>
+    <p>{{_("If this is not possible, you can request a separate census for them to complete.") }}</p>
+    {{
+        onsButton({
+            "text": _("Request separate census"),
+            "classes": 'u-mb-m u-mt-l',
```

Comment:
Our one has too much padding and is missing the `>` in the button. This should be:
```suggestion
            "classes": 'btn u-mb-m u-mt-s btn--link',
```

### Current:

![image](https://user-images.githubusercontent.com/35296336/86611524-b166c480-bfa6-11ea-83b5-cb6139ac89ee.png)

### Expected:

![image](https://user-images.githubusercontent.com/35296336/86611559-ba579600-bfa6-11ea-80c1-75b211033144.png)

Reply:
good spot